src/AccessibilityInsights.SharedUx/Highlighting/Highlighter.cs

```diff
@@ -207,7 +207,7 @@ public void SetCallBackForSnapshot(Action callback)
         /// <returns></returns>
         private static SolidColorBrush GetBrush(HighlighterColor color)
         {
-            return Application.Current.Resources[$"HL{color.ToString()}"] as SolidColorBrush;
+            return Application.Current.Resources[$"HL{color}"] as SolidColorBrush;
```

Comment:
This is where the "HL" gets prefixed. VS flagged the `ToString()` call as superfluous, so I removed it and it works as expected.